bin/build-changelog

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+set -e
+
+if ! command -v gh > /dev/null; then
+  echo "Please install the GitHub CLI: https://cli.github.com/"
+  exit 1
+fi
+
+last_release=$(git describe --tag --abbrev=0)
+
+template_changes=$(git diff --name-only "${last_release}" | grep html.erb)
+
+echo "The following templates have changed since ${last_release}:"
+echo
+echo "$template_changes" | while read -r line; do
+	if case $line in spec*) false;; esac; then
+		echo "  ${line}"
+fi
```

Comment:
Never sure with shellscript but, is this possible?

```suggestion
        fi
```

Reply:
Ooh yeah, you actually can!